ci-operator/config/openshift/installer/openshift-installer-main.yaml

```diff
@@ -1253,6 +1253,26 @@ tests:
       CHECK_MOD_LIST: "false"
     test:
     - ref: go-verify-deps
+- as: shellcheck
+  commands: IS_CONTAINER=TRUE ./hack/shellcheck.sh .
+  container:
+    from: src
+  skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)
+- as: tf-fmt
+  commands: IS_CONTAINER=TRUE ./hack/tf-fmt.sh .
+  container:
+    from: src
+  skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)
+- as: tf-lint
+  commands: IS_CONTAINER=TRUE ./hack/tf-lint.sh .
+  container:
+    from: src
+  skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)
```

Comment:
Let's drop `tf-fmt` & `tf-lint` because no more terraform!

Reply:
@patrickdillon I thought the same thing but aren't some of the other providers like ibm cloud still using it?